src/entrypoints/custom-panel.ts

```diff
@@ -86,7 +88,49 @@ function initialize(
       (err) => {
         // eslint-disable-next-line
         console.error(err, panel);
-        alert(`Unable to load the panel source: ${err}.`);
+        let errorScreen;
+        if (panel.url_path === "hassio") {
+          import("../layouts/supervisor-error-screen");
+          errorScreen = document.createElement(
+            "supervisor-error-screen"
+          ) as any;
+        } else {
+          import("../layouts/hass-error-screen");
+          errorScreen = document.createElement("hass-error-screen") as any;
+          errorScreen.error = `Unable to load the panel source: ${err}.`;
+        }
+        const errorStyle = document.createElement("style");
+        errorStyle.innerHTML = `body{
+          ${
+            (properties.hass as HomeAssistant).themes.darkMode
+              ? `
+              --primary-text-color: #e1e1e1;
+              --primary-background-color: #111111;
+              --card-background-color: #1c1c1c;
+              --app-header-text-color: #e1e1e1;
+              --primary-color: #0288d1;
+              --app-header-background-color: #101e24;`
+              : `
+              --primary-text-color: #212121;
+              --primary-background-color: #fafafa;
+              --card-background-color: #ffffff;
+              --app-header-text-color: #ffffff;
+              --primary-color: #03a9f4;
+              --app-header-background-color: var(--primary-color);`
+          }
+          --mdc-theme-primary: var(--primary-color);
+          --header-height: 56px;
+          font-family: Roboto, sans-serif;
+          display: block;
+          background-color: var(--primary-background-color);
+          color: var(--primary-text-color);
+          height: calc(100vh - 32px);
+          width: 100vw;
+        }`;
```

Comment:
(I know it is a draft)

We should keep these in a const in a file and import them at the multiple places we use them, (onboarding, auth, here, etc)

Reply:
yeah, just put them here to have a quick way to difference while testing